src/pages/Login/LoginPage.jsx

```diff
@@ -83,20 +83,22 @@ export default function LoginPage() {
         spacing={2}
         sx={{ width: "100%", maxWidth: "360px", px: 2 }}
       >
-        <Box
-          component="img"
-          src={Logo}
-          alt="Logo"
-          sx={{
-            width: "80%",
-            maxWidth: "300px",
-            height: "auto",
-            my: 4,
-            "@media (min-width:600px)": {
-              maxWidth: "80%",
-            },
-          }}
-        />
+        <a href="/homepage">
```

Comment:
`/homepage` is not a valid url - did this work on your machine?

Reply:
changes made